src/Components/Categories.jsx

```diff
@@ -19,7 +20,7 @@ return ( (loading) ? <div className="loader">...Loading</div> :
    return <li key={category.slug} className='ReviewCard'>
     <p>Category: {category.slug}</p>
     <p>Description: {category.description}</p>
-   <Link to={`/reviews/${category.slug}`}>Select</Link>
+   <button onClick={()=>navigate(`/reviews/${category.slug}`)}>Select category</button>
```

Comment:
Instead of using useNavigate, we can instead have this button wrapped in a \<Link> tag